Formula/go.rb

```diff
@@ -3,6 +3,10 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
+    # Version bumps of this formula require formulae that depend on Go at
```

Comment:
Having patch version bumped go myself multiple times, these extra revision bumps on dependant formula were not needed. Based on what we know now, I'd recommend clarifying this to just major/minor.